ccloud-exporter.tf

```diff
@@ -29,7 +49,6 @@ resource "kubernetes_secret" "ccloud_exporter_config" {
   data = {
     CCLOUD_API_KEY    = confluent_api_key.ccloud_exporter_api_key[0].id
     CCLOUD_API_SECRET = confluent_api_key.ccloud_exporter_api_key[0].secret
-    CCLOUD_CLUSTER    = confluent_kafka_cluster.cluster.id
```

Comment:
Are you sure about this?  I presume it's not needed since API Keys can be provisioned at the cluster level, but what if they are provisioned at the service account?